### ISSUE TITLE
cspyce fails to install on Python 3.13 Ubuntu

### ISSUE THREAD

**Issue by rfrenchseti** [SETI/rms-cspyce#177]
cspyce fails to install on Python 3.13 and Ubuntu. Oddly it works with MacOS and Windows with Python 3.13.

```
$ pip install cspyce==2.3.0
Collecting cspyce==2.3.0
  Downloading cspyce-2.3.0.tar.gz (59.4 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 59.4/59.4 MB 9.9 MB/s eta 0:00:00
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error
  
  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [37 lines of output]
        % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                       Dload  Upload   Total   Spent    Left  Speed
      
        0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
       66 40.6M   66 27.0M    0     0  28.1M      0  0:00:01 --:--:--  0:00:01 28.1M
      100 40.6M  100 40.6M    0     0  33.2M      0  0:00:01  0:00:01 --:--:-- 33.2M
      <string>:138: DeprecationWarning: Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.
      Traceback (most recent call last):
        File "/tmp/venv/lib/python3.13/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>
          main()
          ~~~~^^
        File "/tmp/venv/lib/python3.13/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
                                   ~~~~^^^^^^^^^^^^^^^^^^^^^^^^
        File "/tmp/venv/lib/python3.13/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 118, in get_requires_for_build_wheel
          return hook(config_settings)
        File "/tmp/pip-build-env-xbeic9uk/overlay/lib/python3.13/site-packages/setuptools/build_meta.py", line 334, in get_requires_for_build_wheel
          return self._get_build_requires(config_settings, requirements=[])
                 ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/tmp/pip-build-env-xbeic9uk/overlay/lib/python3.13/site-packages/setuptools/build_meta.py", line 304, in _get_build_requires
          self.run_setup()
          ~~~~~~~~~~~~~~^^
        File "/tmp/pip-build-env-xbeic9uk/overlay/lib/python3.13/site-packages/setuptools/build_meta.py", line 320, in run_setup
          exec(code, locals())
          ~~~~^^^^^^^^^^^^^^^^
        File "<string>", line 182, in <module>
        File "<string>", line 97, in download
        File "/usr/lib/python3.13/shutil.py", line 593, in copytree
          return _copytree(entries=entries, src=src, dst=dst, symlinks=symlinks,
                           ignore=ignore, copy_function=copy_function,
                           ignore_dangling_symlinks=ignore_dangling_symlinks,
                           dirs_exist_ok=dirs_exist_ok)
        File "/usr/lib/python3.13/shutil.py", line 491, in _copytree
          os.makedirs(dst, exist_ok=dirs_exist_ok)
          ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "<frozen os>", line 227, in makedirs
      FileExistsError: [Errno 17] File exists: 'cspice/Linux-x86_64/src'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.

```

**Comment by fyellin** [comment]
cibuildwheel is now at version v2.22.0.  We use v2.16.5 in out "publish" ymls.  This bug appears to have been fixed in v2.21.0 

"Fix some bugs building linux wheels on macOs".  

I don't really have any easy way of testing this, so I think I need to pass this back to you.

**Comment by rfrenchseti** [comment]
Fixed by #178

**Comment by fyellin** [comment]
I’m glad the fix was as simple as I’d hoped.
